build/base/entrypoint.sh

```diff
@@ -10,17 +10,18 @@ function resolve_host() {
   check="nslookup $host"
   max_retry=10
   counter=0
-  backoff=0.1
+  backoff=1
```

Comment:
```suggestion
  backoff=3
```

@Syed-Suhaan It seems 1 sec is not enough: https://github.com/kubeflow/mpi-operator/actions/runs/20128945073/job/57765870629?pr=757

Let's increase it to 3 sec.

Reply:
@Syed-Suhaan Uhm, 3 sec seems still not enough time. Did 3 sec work well in your local?
If not, we might need to increase it to 5 sec.